models/DataObject/ClassDefinition/Data/Relations/AbstractRelations.php

```diff
@@ -69,9 +74,20 @@ public function setClasses($classes)
      */
     public function getLazyLoading()
     {
-        return true;
+        return $this->lazyLoading;
```

Comment:
why?

Reply:
Doesn't make sense, needs to be reverted 😊